api/camunda/camunda-openapi.yaml

```diff
@@ -10,7 +10,7 @@ info:
     url: https://github.com/camunda/camunda/blob/main/licenses/CAMUNDA-LICENSE-1.0.txt
 externalDocs:
   description: Find out more
-  url: https://docs.camunda.io/docs/apis-tools/camunda-api-rest/camunda-api-rest-overview/
+  url: https://docs.camunda.io/docs/apis-tools/orchestration-cluster-api-rest/camunda-api-rest-overview/
```

Comment:
I am not sure if these yaml files are pointing to current or next...worth double checking if these references should be changed or left alone for now.

Reply:
Very good question, I was also struggling with this a bit. From the OpenAPI file we can point to the released version, I think that is fine.
Here in the docs, it doesn't matter much, I think, as we don't use this information to generate anything, as far as I know.